src/framework/preferences/css/stylus/SeparatedPanelPrefsEditor.styl

```diff
@@ -23,45 +23,55 @@
 
 // Top bar and show/hide button
 .fl-prefsEditor-separatedPanel {
+    display: flex;
+    flex-direction: column;
+
     .fl-panelBar {
         width: 100%;
         background-color: #fff;
         border-bottom: 1px solid #ccc;
+        box-shadow: 0px 2px 3px 0 #6f6f6f;
+        font-size: 1.6rem;
 
-        .fl-prefsEditor-buttons {
-            box-shadow: 2px 2px 3px 0 #6f6f6f;
-            border-bottom: 1px solid #ccc;
-            border-left: 1px solid #ccc;
-            border-right: 1px solid #ccc; //FLUID-4991: The shadow effect around the hide/show preferences bug fix. (Border right added)
-            border-bottom-left-radius:8px;
-            border-bottom-right-radius:8px;
-            background-color: white;
-            min-height: 2em;
-            margin-right: 10em;
-            display: block;
-            float: right;
+        // panelBar for desktop view
+        &.fl-panelBar-below {
+            display: none;
         }
-    }
 
-    .fl-prefsEditor-showHide {
-        min-width: 15em;
+        .fl-prefsEditor-buttons {
+            width: 100%;
+            height: 66px;
```

Comment:
@cindyli yes, so that the button size remains fixed on mobile. When using rem values, it will cause the button to grow and push the sliding panel down as a user is interacting with it. For the other lines, it's to save space when adjusting text size in the panels. I'll add some comments to the styl file.